README.md

```diff
@@ -45,7 +45,7 @@ Production build: `npx react-native run-android --variant=prodDebug`
 
 **iOS:**
 
-Development build: run "npx react-native start" -> select ios -> run at another terminal "react-native run-ios --scheme DoGoodGetGoodDev"
+Development build: run `npx react-native start` -> select ios by pressin "i" -> run at another terminal `react-native run-ios --scheme DoGoodGetGoodDev`
```

Comment:
I'd Change:
`pressin` to `pressing`
`run at another terminal` to `in a new terminal, run`